core/scss/components/brand-bar/_brand-bar--white.scss

```diff
@@ -0,0 +1,12 @@
+@charset "UTF-8";
+
+// Brand bar with cardinal red wordmark on white background
+
+.su-brand-bar--white {
+  background-color: $color-white;
+
+  .su-brand-bar__logo {
+    @include font-smoothing(auto);
+    color: $color-cardinal-red;
+  }
+}
```

Comment:
The white variant looks **really bad** on my large, non-retina monitor at home. See attached image. Is there anything you can do to clean up Stanford's wordmark so it looks excellent even on lower res monitors?

![brand bar non-retina](https://user-images.githubusercontent.com/5298452/47867227-250e0280-ddbe-11e8-87dd-1a122b1400df.png)

Reply:
eek